### PR TITLE
used WorkerRuntime in standalone

### PR DESCRIPTION
**Pull request by ahmet-uyar**
Standalone was giving an error since it was not initializing WorkerRuntime. 
I initialized it. 
I also disabled fault-tolerance and zoo keeper settings in standalone if they are set. 
Since, we are neither supporting fault tolerance nor zoo keeper at standalone yet.

### CODE REVIEW

File: twister2/resource-scheduler/src/java/edu/iu/dsc/tws/rsched/schedulers/standalone/MPIWorker.java [hunk 1]
```diff
@@ -446,7 +457,8 @@ private void startWorker(Config cfg, int rank, Intracomm intracomm, JobAPI.Job j
       initLogger(cfg, intracomm.getRank(), twister2Home);
 
       // now create the worker
-      IWorkerController wc = createWorkerController(job);
+//      IWorkerController wc = createWorkerController(job);
```
Comment (supunkamburugamuve) [inline twister2/resource-scheduler/src/java/edu/iu/dsc/tws/rsched/schedulers/standalone/MPIWorker.java]: As discussed, I'm not sure whether abandoning this method completely is correct

Comment (ahmet-uyar) [reply twister2/resource-scheduler/src/java/edu/iu/dsc/tws/rsched/schedulers/standalone/MPIWorker.java]: I tested all three cases with new commits: no JM, JM at client, JM at worker 0. 
All seems to be working.

**Comment by supunkamburugamuve** [comment]
@ahmet-uyar there is a conflict

**Comment by ahmet-uyar** [comment]
> @ahmet-uyar there is a conflict

fixed it. it was because of the other merged pull request.

**Pull request merged**